docs/CHANGELOG.asciidoc

```diff
@@ -40,6 +41,78 @@ Review important information about the {kib} 8.7.0 release.
 //* <<release-notes-8.0.0-alpha2>>
 //* <<release-notes-8.0.0-alpha1>>
 --
+[[release-notes-8.7.1]]
+== {kib} 8.7.1
+
+coming::[8.7.1]
+
+Review the following information about the {kib} 8.7.1 release.
+
+[float]
+[[breaking-changes-8.7.1]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 8.7.1, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+There are no breaking changes in the {kib} 8.7.1 release.
+// end::notable-breaking-changes[]
+
+To review the breaking changes in the previous release, check {kibana-ref-all}/8.7/release-notes-8.7.0.html#breaking-changes-8.7.0[8.7.0].
+
+[float]
+[[enhancement-v8.7.1]]
+=== Enhancement
+Fleet::
+The agent policy "Host name format" selector is now enabled by default {kibana-pull}154563[#154563]
+
+[float]
+[[fixes-v8.7.1]]
+=== Bug fixes
+APM::
+* Scoring is now applied by ES {kibana-pull}154627[#154627]
+* Fixes the APM Java Agent download link {kibana-pull}154023[#154023]
+* Improves the overflow message text {kibana-pull}153676[#153676]
+
+Canvas::
+* Disables the Edit in Lens action for the legacy savedVisualization function {kibana-pull}154656[#154656]
+* Fixes the home page redirect loop {kibana-pull}154568[#154568]
+* Fixes an issue where the image upload component was unable to load for image elements {kibana-pull}154385[#154385]
+
+Dashboard::
+Improves the async controls flyout behaviour {kibana-pull}154004[#154004]
```

Comment:
We could re-word this to "Improves controls flyout performance for data views with a large number of fields."